src/runtime.js

```diff
@@ -22,6 +27,15 @@ let tornado = {
   },
 
   /**
+   * Register a helper, overwrite if a helper already exists
+   * @param {String} name The name of the helper
+   * @param {Function} method The function to be executed when the helper is found in a template
+   */
+  registerHelper(name, method) {
```

Comment:
A formal register helper is nice, but if the helpers object is still public, we will run into the same problems as Dust with blowing away the previous helpers by accident.